conans/test/integration/command_v2/test_combined_pkglist_flows.py

```diff
@@ -162,3 +162,51 @@ def test_download_upload_only_recipes(self, client, prev_list):
         assert f"Uploading recipe 'zli/" not in client.out
         assert "Uploading package 'zlib/1.0.0" not in client.out
         assert "Uploading package 'zli/" not in client.out
+
+
+class TestListRemove:
+    @pytest.fixture()
+    def client(self):
+        c = TestClient(default_server_user=True)
+        c.save({
+            "zlib.py": GenConanfile("zlib"),
+            "zli.py": GenConanfile("zli", "1.0.0")
+        })
+        c.run("create zli.py")
+        c.run("create zlib.py --version=1.0.0 --user=user --channel=channel")
+        c.run("upload * -r=default -c")
+        return c
+
+    def test_remove_nothing_only_refs(self, client):
+        # It is necessary to do *#* for actually removing something
+        client.run(f"list * --format=json", redirect_stdout="pkglist.json")
```

Comment:
We have discussed about this, and it seems ok. 
It will require explaining it in the docs, but this is expected and aligned with the ``conan remove`` command defaults, which are way more important.